### ISSUE TITLE
Cooking Daily Quest Icon Issue

### ISSUE THREAD

**Issue by TheHulkingMango** [Questie/Questie#4825]
Issue Report
---------------------
When picking up the Mustard for the Cooking Daily "[80] Mustard Dogs!", the reagent counter that was recently added replaces the "Empty Picnic Basket" is not tied to/does not allow for clicking on the reagent to craft the "Mustard Dog Picnic Basket" as it used to. Bug/Issue was encountered while completing the Dalaran Cooking Daily, I'm unsure if it effects other quests that are of this style where a reagent is gathered and combined with another item in the inventory to create the required quest item.

Questie Version: 8.0.1

WoW Version: WotLK 

![image](https://user-images.githubusercontent.com/132087348/235201842-942ca362-a2bb-4648-b8b1-c5e48120c9f1.png)

![image](https://user-images.githubusercontent.com/132087348/235202082-cec47cf8-9134-4205-a993-95cdc9388907.png)

**Comment by Cabro** [comment]
Same for Crystalsong Carrots. For some reason those items get picked up by Questie as items to be used.

**Comment by Dyaxler** [comment]
It's totally ok for me... *scratches head*

![image](https://user-images.githubusercontent.com/17400948/235235077-bdb50973-584a-47c1-80e4-3be7070a7eef.png)

**Comment by Dyaxler** [comment]
@TheHulkingMango Wait... I just re-read your opening post. Are you saying that the Wild Mustard texture replaced the Empty Picnic Basket, _but the original tooltip remained_?

**Comment by TheHulkingMango** [comment]
No, when you have the Wild Mustard and the Basket in the inventory Questie used to show the basket next to the quest allowing you to click on the icon to create the picnic basket. Now it shows the Wild Mustard and clicking on it does nothing. You have to go into the inventory to click on the basket itself.

**Comment by Dyaxler** [comment]
I got the Mustard Dog one again... still no issues.

![image](https://user-images.githubusercontent.com/17400948/235363173-c76a5a90-bf08-4e0d-a925-cd8b29fada54.png)

**Comment by TheHulkingMango** [comment]
I got the Mustard Dog quest as well and it seems to be fixed. The basket showed next to the quest in Questie and was clickable outside of the inventory. The Wild Mustard didn't replace the Basket Icon forcing the user to open their inventory to click on the Mustard Dog Picnic Basket and complete the quest.

**Comment by Dyaxler** [comment]
Hmmm... weird. Wonder why it happened to begin with. I swear I didn't touch this part of the code base since v8. :D

**Comment by Dyaxler** [comment]
@Cabro as discussed on Discord. Found the bug and fixed it.
